### DISCUSSION TITLE
Replace URLs with display text

### DISCUSSION THREAD

**Discussion by Liquidmantis** [nvim-telekasten/telekasten.nvim#27] [Ideas]
In vimwiki an external link is replaced with its display text rather than the full markdown link syntax when the line isn't selected.  It's expanded when the line is active for visibility and editing.

For example:
`[My cool link](http://bitly.com/98K8eH)` would just show as `_My cool link_` when the line isn't the current line.

**Comment by renerocksai** [comment]
This looks dangerous :smile: but it is worth a try. Ideally, I'd like to use virtual text / extmarks for that but as described elsewhere they might not be the best fit.
